docs/sources/integrations/_index.md

```diff
@@ -18,17 +21,15 @@ An "Integration" is a main entry point for alerts being consumed by Grafana OnCa
 Integrations receive alerts on a unique API URL, interprets them using a set of templates tailored for the monitoring system, and starts
 escalations.
 
-For more information about the templating used in OnCall, refer to [Jinja2 templating][].
-
 ## Learn Alert Flow Within Integration
 
 1. An Alert is received on an integration's **Unique URL** as an HTTP POST request with a JSON payload (or via
 [Inbound email][], for inbound email integrations)
-1. Routing is determined for the incoming alert, by applying the [Routing Template][]
-1. Alert Grouping is determined based on [Grouping Id Template][]
```

Comment:
also fixed this issue here:
![Screenshot 2024-06-17 at 11 44 13](https://github.com/grafana/oncall/assets/9406895/f863dad0-1528-4960-a509-c2059d7b5147)